Resources/Views/Home/_speakers.leaf

```diff
@@ -24,7 +24,7 @@
         #for(speaker in speakers):
             <div class="speaker-grid-element">
                 <div class="speaker-grid-element-image-container">
-                    <img class="speaker-grid-element-image" src="https://swiftleeds-speakers.s3.eu-west-2.amazonaws.com/#(speaker.profileImage)"/>
+                    <img class="speaker-grid-element-image" src="https://swift-leeds.s3.eu-west-2.amazonaws.com/#(speaker.profileImage)"/>
```

Comment:
Whoops. This is my local bucket. Will swap it back before it gets merged in.

Reply:
I've changed back but GitHub isn't seeing this as an outdated comment for some reason. Will resolve conversation instead.